slack/channels.py

```diff
@@ -49,6 +50,40 @@ def mark(channel, ts):
 
     return slack.http_client.post('channels.mark', data)
 
+def create(name):
+    """
+    Creates a new channel.
+    """
+    data = {
```

Comment:
I think we've been naming this `params` instead of `data`. Would you mind keeping it consistent?